src/js/lightgallery.js

```diff
@@ -460,6 +460,12 @@
                 subHtmlUrl = this.$items.eq(index).attr('data-sub-html-url');
             } else {
                 subHtml = this.$items.eq(index).attr('data-sub-html');
+                if (!subHtml) {
+                    subHtml = this.$items.eq(index).attr('title') || this.$items.eq(index).find('img').attr('alt');
```

Comment:
> And how do we manage if there are multiple images under a selector?
jQuery get first attribute element from collection. And multiple thumb-images its possible?